app/jobs/create_derivatives_job_decorator.rb

```diff
@@ -11,7 +11,15 @@ module CreateDerivativesJobDecorator
     def perform(file_set, file_id, filepath = nil)
       return if file_set.video? && !Hyrax.config.enable_ffmpeg
       # OVERRIDE HYRAX 3.4.1 to skip derivative job if rdf_type is "pcdm-muse:PreservationFile"
-      return if file_set.rdf_type.downcase.include? PRESERVATION_FILE
+      @remaining_retries ||= 2
```

Comment:
👍🏾

Reply:
This will not work as desired.

Each time we call `CreateDerivativesJob.perform` we instantiate a new object; note the `def perform` is an instance method and `CreateDerivativesJob.perform` is a class method call.  This means the instance variable will always be 2.

What we need is to pass this as a parameter.

Here is an example of that implementation: https://gitlab.com/notch8/adventist-dl/-/blob/c924c8aed10d354d2c4992981b0905ef15a120f3/app/jobs/index_plain_text_files_job.rb